posts/2023-09-01-electing-new-project-directors.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Electing New Project Directors"
+author: Leadership Council
+team: Leadership Council <https://www.rust-lang.org/governance/teams/leadership-council>
+---
+
+Today we are launching the process to elect new Project Directors to the Rust Foundation Board of Directors.
+As we begin the process, we wanted to spend some time explaining the goals and procedures we will follow.
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.
+These Project Directors serve as representatives of the Rust project itself on the Board.
+Like all Directors, the Project Directors are elected by the entity they represent, which in the case of the Rust Project means they are elected by the Rust Leadership Council.
+Project Directors serve for a term of two years and will have staggered terms.
+This year we will appoint two new directors and next year we will appoint three new directors.
+
+We want to make sure the Project Directors can effectively represent the project as a whole, so we are soliciting input from the whole project.
+The elections process will go through two phases: Candidate Gathering and Election.
+Read on for more detail about how these work.
+
+## Candidate Gathering
+
+The first phase is beginning right now.
+In this phase, we are inviting the members of the Rust Project to nominate people who will make good project directors.
+The goal is to bubble these up to the Council through each of the top-level teams.
+You should be hearing from your Council Representative soon with more details, but if not, feel free to reach out to them directly.
+
+Each team is encouraged to suggest candidates.
+Since we are electing two new directors, it would be ideal for teams to nominate two candidates, but teams can nominate more or fewer as they see fit.

```

Comment:
This feels iffy to me -- I think we should say *at least* two candidates. We're somewhat worried about not having enough candidates, I definitely don't want a team to think two is a quota they shouldn't exceed.

Reply:
Sounds good. I changed it to say at least two, and removed the part that said more or fewer is okay. I figure since it still says "it's ideal" we're not making it a requirement, but we're also not highlighting the fact that teams aren't required to nominate anyone.